mandatory/3-playing-computer.js

```diff
@@ -20,10 +20,12 @@ let a = 6;
 
 const f1 = function (a, b) {
   return a + b;
+  return a + b;
 };
 
 const f2 = function (a, b) {
   return a + b + x;
+  return a + b + x;
 };
 
 console.log(x);
```

Comment:
Hi Sandra good job. Just want to understand what you want to see when you console log x?